charts/ingress/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.4.0
+
+### Improvements
+
+- Generate the `adminApiService.name` value from `.Release.Name` rather than 
+  hardcoding to `kong`
+  [#839](https://github.com/Kong/charts/pull/839)
```

Comment:
This would also need to be mentioned in `kong` chart right? And that would need to be bumped as well.

I believe a better course of action for this would be to first introduce this in `kong` get that released and only after use it in `ingress` chart. WDYT?

Reply:
This PR changes the `kong` chart (I forgot to bump the version, doing so now) _and_ updates the `values.yaml` default in `ingress`. Both should go out together

Reply:
Ah, once `kong` is released we need to update `Chart.lock` in `Ingress`. I'll split the work